src/components/common/SearchFiltersWrapper.tsx

```diff
@@ -63,15 +81,20 @@ export default function Search({ search, onSearch, filtersProps }: SearchProps):
             const pillValue =
               filtersProps.pillValuesRenderer && filtersProps.pillValuesRenderer(key, filtersProps.filters[key].values);
 
+            let label = filtersProps.filterColumns.find((f) => key === f.name)?.label ?? '';
+            if (!label) {
```

Comment:
This allows for the pill list to be populated by the featured filters and the regular filters